dags/analyzer_helper/discourse/transform_raw_info.py

```diff
@@ -0,0 +1,93 @@
+from analyzer_helper.discourse.utils.convert_date_time_formats import DateTimeFormatConverter
+
+class TransformRawInfo:
+
+    def __init__(self):
+        self.converter = DateTimeFormatConverter()
+    
+    def create_data_entry(self, raw_data: list, interaction_type: str = None, interaction_user: int = None):
+        metadata = {
+            "channel_id": raw_data.get("category_id"),
+            "thread_id": raw_data.get("topic_id"),
```

Comment:
Please use the same `topic_id` instead of `thread_id`. So this would be 
`"topic_id": raw_data.get("topic_id"),`